src/main/java/com/commercetools/sync/categories/CategorySync.java

```diff
@@ -434,29 +434,29 @@ private void processFetchedCategories(@Nonnull final Set<Category> fetchedCatego
 
     /**
      * Given a {@link Set} of categories and a {@code key}. This method tries to find a category with this key in this
```

Comment:
```suggestion
     * Given a {@link Set} of categories and a {@code key}, this method tries to find a category with this key in this
```
https://www.grammarly.com/blog/commas-after-introductory-phrases/